### ISSUE TITLE
Order by Date in Table

### ISSUE THREAD

**Issue by veryrobert** [distantnative/retour-for-kirby#344]
## Order redirects by date added in table

If you have a large list of redirects it becomes extremely hard to follow and track what and when was added if you can order by date in the table view.

**Comment by distantnative** [comment]
So you want to order by creation date not by date when the redirect was last used?

**Comment by veryrobert** [comment]
Yes, most of the time when we use it, we have a list of redirects that need to be moved across. After you add them, they disappear instead of being viewable by date created. So, I typically view the .yml file to cross-check where I’m at. It becomes more difficult when moving across environments or if a client is handling the redirects themselves.

**Comment by distantnative** [comment]
> After you add them, they disappear

They aren't in the list anymore? Can you provide a screencast so I better understand the indue, please?

**Comment by veryrobert** [comment]
> > After you add them, they disappear
> 
> They aren't in the list anymore? Can you provide a screencast so I better understand the indue, please?

Sorry, they are in the list – but as you can't order by date created it's hard to see where you have left off if you have a considerable amount of redirects to go through:

https://capture.dropbox.com/yKUFt0L8BB9ZTzJo
